dist/widgets/WalletModal/useWalletModal.d.ts

```diff
@@ -1,7 +1,6 @@
 import { Login } from "./types";
 interface ReturnType {
     onPresentConnectModal: () => void;
-    onPresentConnectModal1: () => void;
```

Comment:
This was most likely used while building the project and is no longer needed. Removed from types as well